Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:36 AS builder
+FROM registry.fedoraproject.org/fedora:37 AS builder
```

Comment:
This needs to update the second `FROM` below too.

Reply:
Awesome catch thank you, and ok sorry I was going off of the butane train, didnt look on all apps; assumed it was a standard not to mention it.